doc/factory/configuration.html

```diff
@@ -964,6 +967,55 @@ <h3 class="western"><a name="args_global"></a>Global arguments</h3>
               <a href="custom_scripts.html">custom scripts page</a>.
             </p>
           </li>
+          <li>
+            <a name="cvmfsexec_distro" />
+            <div class="xml">
+              &lt;glidein&gt;&lt;cvmfsexec_distro
+              sources=&quot;<i>sources_list</i>&quot;
+              platforms=&quot;<i>platforms_list</i>&quot; /&gt;
+            </div>
+
+            <p>
+              This knob provides control over the building and/or rebuilding of
+              cvmfsexec distributions when CVMFS is to be provisioned on demand.
+              Distributions built via this knob will be located in factory work
+              directory location (by default, for an RPM-based install of
+              GlideinWMS, this would typically be
+              <i>/var/lib/gwms-factory/work-dir</i>). Two attributes are
+              available to control the behavior of this knob and are described
+              below. If both the attributes are empty, then (re)building of
+              cvmfsexec distributions will be disabled (the default behavior).
+            </p>
+            <p>
+              The <b><i>sources</i></b> attribute specifies the source(s) to
+              download the latest cvmfs configuration and repositories from.
+              Must be at least one value or a comma-separated list of values
+              from the options { <i>osg</i> | <i>egi</i> | <i>default</i> }.
+            </p>
+            <p>
+              The <b><i>platforms</i></b> attribute indicates machine types
+              (platform- and architecture-based) for which distributions is to
+              be built. Can be empty, a single value or a comma-separated list
+              of values from the options { <i>rhel7-x86_64</i> |
+              <i>rhel8-x86_64</i> | <i>suse15-x86_64</i> }.
+            </p>
+            <p>
+              Note that, <b><i>sources</i></b> is a <b>required</b> attribute
+              and <b><i>platforms</i></b> is an <b>optional</b> attribute.
+              <li>
+                if <b><i>platforms</i></b> is empty and <b><i>sources</i></b> is
+                non-empty), the attribute value for <b><i>platforms</i></b> will
+                default to the string
+                <i>rhel7-x86_64,rhel8-x86_64,suse15-x86_64</i>.
```

Comment:
Tell that the default is all available platforms:
```
... is non-empty), the distribution will be built for all available platforms (<i>rhel7-x86_64,rhel8-x86_64,suse15-x86_64</i>).
```

Reply:
rephrased as suggested.